codebuild/bin/install_ubuntu_dependencies.sh

```diff
@@ -18,6 +18,9 @@
 
 set -ex
 
+# Update Yarn Debian key, see https://github.com/yarnpkg/yarn/issues/7866
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg  | sudo apt-key add -
```

Comment:
Is it safe to trust this key?

Reply:
It is as safe as trusting the listed sigh to deliver the binary packages we're installing over TLS. If they (and their site) are already within our trust boundary, then yes.